packages/big-design/src/components/Worksheet/spec.tsx

```diff
@@ -287,6 +283,34 @@ describe('edition', () => {
     await waitForElement(() => screen.getAllByRole('combobox'));
   });
 
+  test('regains focus when it stops editing', async () => {
+    const { getByText, getByDisplayValue, getByTestId } = render(
+      <Worksheet columns={columns} items={items} onChange={handleChange} />,
+    );
+
+    const table = getByTestId('worksheet');
```

Comment:
Alongside @rtalvarez comments around using `screen`, could we not use `data-testid` here? I think something like `screen.getByRole('table')` should:tm: work.